pages/task/read/routers/view.js

```diff
@@ -8,7 +8,12 @@ const { cleanModel } = require('../../../../lib/utils');
 const getContent = require('../content');
 const { getNacwoById, getEstablishment } = require('../../../common/helpers');
 
-const getRelevantActivity = activityLog => activityLog.filter(log => {
+const getRelevantActivity = activityLog => activityLog.filter((log, index) => {
+  if (get(log, 'event.meta.payload.data.extended') && index > 0) {
```

Comment:
why are we checking index > 0? I'd either check for something more explicit or add a comment

Reply:
If extending the deadline was the most recent activity then we don't show it in the activity log, we only show it in the log once the task status changes.

Reply:
I think that's an old requirement that we got rid of. See https://jira.digital.homeoffice.gov.uk/browse/AS-1231

Reply:
Ah, the design and the ticket had conflicting info - I'll remove the check.

Reply:
Sorted.